pkg/cmd/util/docker/docker.go

```diff
@@ -36,6 +38,34 @@ func (_ *Helper) GetClient() (client *docker.Client, endpoint string, err error)
 	return
 }
 
+// KubeDocker provides a wrapper to Kubernetes Docker interface
+// This wrapper is compatible with OpenShift Docker interface.
+type KubeDocker struct {
+	dockertools.DockerInterface
+}
+
+// Ping implements the DockerInterface Ping method.
+func (c *KubeDocker) Ping() error {
+	client, err := docker.NewClientFromEnv()
+	if err != nil {
+		return err
+	}
+	return client.Ping()
+}
+
+// GetKubeClient returns the Kubernetes Docker client.
+func (_ *Helper) GetKubeClient() (*KubeDocker, string, error) {
+	var endpoint string
+	if len(os.Getenv("DOCKER_HOST")) > 0 {
+		endpoint = os.Getenv("DOCKER_HOST")
+	} else {
+		endpoint = "unix:///var/run/docker.sock"
+	}
+	client := dockertools.ConnectToDockerOrDie(endpoint)
```

Comment:
@liggitt I will have to export the getDockerClient in kube to make that work... I will do that as a follow up if you are OK with it.